### PR TITLE
feat: add impact section

### PR DESCRIPTION
**Pull request by yansal**

### CODE REVIEW

File: content/docs/ulule-api/reference/impact.md [hunk 1]
```diff
@@ -0,0 +1,82 @@
...
+| `id`          | int                           | Unique ID of the project SDG      |
+| `sdg`         | [SDG resource](#sdg-resource) | SDG                               |
+
+### SDG resource
+
+| Field         | Type                                    | Description                                                            |
+| ------------- | --------------------------------------- | ---------------------------------------------------------------------- |
+| `code`        | string                                  | Code of the SDG                                                        |
+| `description` | [i18n object](#i18n)                    | Description of the SDG                                                 |
+| `logo`        | [image resource](#image)                | Logo image of the SDG                                                  |
```
Comment (edouardparis) [inline content/docs/ulule-api/reference/impact.md:33]: logo may be i18n

File: content/docs/ulule-api/reference/impact.md [hunk 2]
```diff
@@ -0,0 +1,82 @@
+---
+title: "Impact"
+weight: 11
+---
+
+# Impact
```
Comment (edouardparis) [inline content/docs/ulule-api/reference/impact.md:6]: maybe add link to the un.org

Comment (yansal) [reply content/docs/ulule-api/reference/impact.md:6]: done

File: content/docs/ulule-api/reference/impact.md (hunk 1 above)
Comment (yansal) [reply content/docs/ulule-api/reference/impact.md]: done

**Pull request merged**